### DISCUSSION TITLE
Relationmanagers in tabs?

### DISCUSSION THREAD

**Discussion by gavinhewitt** [filamentphp/filament#309] [Ideas]
I have resources for which it makes sense to have the form split up in tabs. These resources also have quite a lot of relations. There is currently no way (or not that I know of) to embed the relationmanagers within tabs instead of each manager listed separately on the page. 

is there a way this could be achieved? Or could this be something that's configurable in the future?

**Comment by ryangjchandler** [comment]
@gavinhewitt Resource managers are rendered independently on the form, since they're not technically "field types". This is something we _could_ look at in the future, but we've not actively discussed it.

**Reply by bernhardh** [reply]
I support gavinhewitt's suggestion and great that it's option for you! I really love how you react to suggestions :)

Imho, the user of an admin doesn't care, if something is technically a field or not. Fast access and easy overview to these relations - especially if there are more then one - would be king.

As far as I understand, this could be made of course optional.

**Reply by ryangjchandler** [reply]
@bernhardh Yeah, it's definitely something we can do. My comment regarding it not being a field type is more from an internals point of view, there would need to be quite a few refactors needed for this to work correctly.

**Comment by vv-ac** [comment]
Any update on this?
I would love to place different relationmanagers on different positions in my forms. Having tabs, the table simply sits below everything else, which is managable if it is "just" one relationship but not "fancy".

**Reply by michaelklopf** [reply]
Hey, I'd like to place the relation manager in my infolist to place the resources information around the relationships (in this case a discussion for a product, where the discussion is the core information and the product information should be placed around it for support).

**Comment by chrisscalzo** [comment]
This is an old thread but specific to what I'm trying to solve for. Does Filament allow control of where the individual relationship manager appears on a Form (ie within a Tab, within a specific section, etc.)?

**Reply by bernhardh** [reply]
You still can't put the relations into your form (which is a missing feature I think), but you can do other stuff like grouping it. See documentation, for example https://filamentphp.com/docs/3.x/panels/resources/relation-managers#grouping-relation-managers

**Reply by ryangjchandler** [reply]
@bernhardh You _can_ put fields that modify relationship data (for belongsto/hasone), but having the RM inside of the form itself can be tricky because you can nest `<form>` elements.

**Reply by bernhardh** [reply]
Thanks for feedback!
I know this very nice feature, but I was mainly talking about the RM and I totally understand the problem of nesting forms. I am wondering how Laravel nova solves this issue, since there you can do it as far as I can remember. But that's probably because a completely other architecture.

**Comment by mahfoudfx** [comment]
Hello,
The creator of Filament explain [here](https://www.answeroverflow.com/m/1123606988866191442) why this is not available as a core feature.
But I found a plugin that could be a solution: [Filament Relation Manager Component](https://github.com/njxqlus/filament-relation-manager-component).

**Reply by RaulJJNazca** [reply]
Hola.
Probé este componente y visualmente funciona bien, pero cuando doy al botón "save" del formulario principal no detecta el evento.
No veo el error en las herramientas de desarrollo ,  no hay error javascript pero tampoco hace nada.
